lib/tag-field.jsx

```diff
@@ -194,6 +196,7 @@ const mapStateToProps = ( {
 	const note = getNote( state );
 	const revision = selectedRevision || note;
 	return {
+		allTags: state.tags.map( property( 'data.name' ) ),
```

Comment:
Yep, just be careful about working too far ahead of other existing work. It tends to not save us as much time as we might think it could.